CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 Changelog
 =========
 
+## TBD
+
+### Bug fixes
+
+* Release process now updates BugsnagPerformanceSwift podspec rather than BugsnagPerformanceSwiftUI (which has been deprecated)
```

Comment:
I'm not sure if it should be in the Changelog?

Reply:
The issue would have negatively impacted users who tried to use pod `BugsnagPerformanceSwift`, so it would be important to let them know.